src/components/LottieQR.jsx

```diff
@@ -0,0 +1,15 @@
+import Lottie from 'lottie-react';
+import animationData from '../../public/resources/lottieQR.json';
+
+export const MyLottieAnimation = () => {
```

Comment:
Wooow! I didn't know it was this easy to use Lottie! Great job! I love that you didn't let the animation loop, that tends to make the page a bit messy.